recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 6ff5f3180870836cae40f06fa10419f557208175f13ad7bc26caa77beb1f6e02
 
 build:
-  number: 0
-  script: pip install .
+  number: 1
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Does `python -m` work well here? IIRC it caused some problems before.

Reply:
I have not experienced problems with using python -m, but I am happy to drop that part if it will get this accepted.